src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java

```diff
@@ -237,6 +240,12 @@ public Response listContainer(@NotNull @PathParam("container") String container,
     @XmlRootElement(name = "object")
     @XmlType
     static class ObjectEntry {
+        static class SwiftDateSerializer extends DateSerializer {
```

Comment:
Would this be useful outside of the container resource? Do other resources have dates? I think Containers have their creation date.